### PR TITLE
Miscellaneous fixes to CWP author list

### LINKED ISSUE #48: Potential author list updates for CWP doc
we asked for and got some feedback by checking the reco-trigger author list - these changes might be of interest for the full CWP list as well

EPFL: Institute of Physics, École Polytechnique F\'ed\'erale de Lausanne (EPFL), Lausanne, Switzerland

ETHZurich: ETH Z\"urich - Institute for Particle Physics and Astrophysics (IPA), Z\"urich, Switzerland

LMU: Fakult\"at f\"ur Physik, Ludwig-Maximilians-Universit\"at M\"unchen, M\"unchen, Germany

MIT: Laboratory for Nuclear Science, Massachusetts Institute of Technology, Cambridge, MA, USA
(MIT is different from UMass)

at least some NIKHEF authors are affiliated with Vrije university not university of Amsterdam (ymmv here)
NIKHEF: Nikhef National Institute for Subatomic Physics and Vrije Universiteit Amsterdam, Amsterdam, The Netherlands

UZurich: Physik-Institut, Universit\"at Z\"urich, Z\"urich, Switzerland

### PR DESCRIPTION
**Pull request by jouvin**
Contributes to fixing #48.

### CODE REVIEW

**Comment by jouvin** [comment]
I think this PR addresses #48 but I keep it marked as `WIP` for now until clarification by @davidlange6 on the issue he mentioned with the spelling of some affiliations (see #48 issue).

File: CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt [hunk 1]
```diff
@@ -148,10 +148,10 @@ Kcira, Dorian - Caltech
 Kersevan, Borut Paul - ULjubljana
 Kirby, Michael - Fermilab (3)
 Klimentov, Alexei - BNL
-Klute, Markus - MIT
+Klute, Markus - UMass
 Komarov, Ilya - INFNTRIESTE (14)
 Konstantinov, Dmitri - Protvino
-Koppenburg, Patrick - NIKHEF
+Koppenburg, Patrick - NIKHEF & VU
```
Comment (eduardo-rodrigues) [inline CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt]: Gerhard Raven is indeed Nikhef and VU but Patrick is only employed by Nikhef. I've just checked this on the LHCb authors list to be sure.

File: CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt [hunk 2]
```diff
@@ -267,7 +267,7 @@ Stewart, Gordon - UGlasgow
 Stewart, Graeme - CERN - HSF (1)
 Taffard, Anyes - UCIrvine
 Tamponi, Umberto - INFNTorino
-Templon, Jeff - NIKHEF
+Templon, Jeff - NIKHEF & VU
```
Comment (eduardo-rodrigues) [inline CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt]: Jeff is also only Nikhef.

File: CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt [hunk 3]
```diff
@@ -276,7 +276,7 @@ Valassi, Andrea - CERN
 Vallecorsa, Sofia - UGangneung
 Valsan, Liviu - CERN
 Van Gemmeren, Peter - ANL
-Vázquez Sierra, Carlos - NIKHEF
+Vázquez Sierra, Carlos - NIKHEF & VU
```
Comment (eduardo-rodrigues) [inline CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt]: Only Nikhef.

**Comment by jouvin** [comment]
@eduardo-rodrigues should be ok now for NIKHEF affilitations. Still waiting @davidlange6 clarifications for the problems mentioned in the description...

**Comment by eduardo-rodrigues** [comment]
@jouvin, seems you removed my commits, which in particular added Mike Williams to the authors list?

**Comment by jouvin** [comment]
@eduardo-rodrigues yes I sent you an email about it... Sorry for the mistake but the workflow you used is error prone IMO. Probably the easiest is to accept this PR after removing 'Fixes #48' from the description (or change it into 'contributes to #48') and open a new PR based on the updated master. We can sort out the last details of #48 in a future PR if there is something left to be done in @davidlange6 description.

**Comment by eduardo-rodrigues** [comment]
You are right. Sorry, it was me trying to finish too many things last night :S!
OK, what you suggest sounds very reasonable.

**Pull request merged**